micronaut-maven-plugin/src/main/java/io/micronaut/maven/openapi/OpenApiGenericMojo.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.maven.openapi;
+
+import io.micronaut.openapi.generator.GeneratorOptionsBuilder;
+import io.micronaut.openapi.generator.MicronautCodeGenerator;
+import io.micronaut.openapi.generator.MicronautCodeGeneratorBuilder;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Locale;
+import java.util.Map;
+
+@Mojo(name = OpenApiGenericMojo.MOJO_NAME, defaultPhase = LifecyclePhase.GENERATE_SOURCES)
+public class OpenApiGenericMojo extends AbstractOpenApiMojo {
+    public static final String MOJO_NAME = "generate-openapi-sources";
+    public static final String CONFIGURATION_PROPERTIES = MICRONAUT_OPENAPI_PREFIX + ".generator.properties";
+
+    @Parameter(property = MICRONAUT_OPENAPI_PREFIX + ".generator.builder.classname")
+    protected String generatorClassName;
```

Comment:
Please add Javadoc for all paremeters, in this and the parent abstract mojo (I overlooked those)

They go into the documentation, and now it's empty:

![image](https://github.com/micronaut-projects/micronaut-maven-plugin/assets/153880/ab684e0e-e391-4fe0-8546-3c6bfc02a003)